packages/core/package.json

```diff
@@ -278,6 +278,11 @@
                     "markdownDescription": "%AWS.configuration.description.amazonq.shareContentWithAWS%",
                     "default": true,
                     "scope": "application"
+                },
+                "aws.threatComposer.defaultEditor": {
```

Comment:
as [mentioned](https://github.com/aws/aws-toolkit-vscode-staging/pull/1588/#discussion_r1613951670) , this new setting should not be needed